lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -88,7 +88,7 @@
                             tabs: [{
                                 title: gettext('My Teams'),
                                 url: 'teams',
-                                view: new TempTabView({text: 'This is the new Teams tab.'})
+                                view: new TempTabView({text: '<p class="temp-tab-view">This is the new Teams tab.</p>'})
```

Comment:
Note: I made these changes in order to create some tab-specific CSS for the `#teams` route. Before, the only CSS which I could assert present in the tests was `.page-content-main`, which is present on every teams page. Just asserting that that's present wouldn't test anything, and wouldn't fail when the `TempTabView` is replaced.